src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -130,6 +136,112 @@ private bool TryAuthenticateFromCache(
 		return true;
 	}
 
+	private async Task<IOktaAuthenticatedClient?> TryAuthenticateWithDSSOAsync(
```

Comment:
Not throwing any exceptions here since I think we'd always want to fall back on prompting for password. 

Wondering if maybe there are some setups where the warning messages could be annoying for people. Jumpbox is probably one of them? Was thinking we could put a new value in the config that disables passwordless which by default would be false?

Reply:
Experimental would stay as an arg only option

Reply:
I suppose we have to add this as otherwise jumpbox users will be forced to wait for the timeout